lib/railway_ipc/persistence/published_message.ex

```diff
@@ -31,7 +31,7 @@ defmodule RailwayIpc.Persistence.PublishedMessage do
     |> validate_required([
       :message_type,
       :encoded_message,
-      :exchange,
+      # :exchange,
```

Comment:
Now we will persist messages that are published directly to a queue so any such messages will not have an exchange.